ikarus/assembler/assemblermanipulator.hh

```diff
@@ -0,0 +1,39 @@
+// SPDX-FileCopyrightText: 2021-2024 The Ikarus Developers mueller@ibb.uni-stuttgart.de
+// SPDX-License-Identifier: LGPL-3.0-or-later
+
+/**
+ * \file assemblermanipulator.hh
+ * \brief Defines a decorator for the assemblers that helps to manipulate the assembled quantities
+ */
+
+#pragma once
+
+#include <ikarus/assembler/wrapperinheritancedecider.hh>
+
+namespace Ikarus {
+/**
+ * \class AssemblerManipulator
+ * \brief The AssemblerManipulator defines a decorator for the assemblers that
+ * helps to manipulate the assembled quantities.
+ * \ingroup assembler
+ *
+ * \tparam A Type of the assembler.
```

Comment:
Type of the wrapped assembler

Reply:
Isn't the `AssemblerManipulator` the wrapper that wraps and assembler. So, here it is the assembler that is being wrapped?

Reply:
Yes A is the wrapped assembler which is wrapped by the AssemblerManipulator